source/host_resolver.c

```diff
@@ -20,6 +20,7 @@
 #include <inttypes.h>
 
 const uint64_t NS_PER_SEC = 1000000000;
+const size_t AWS_DEFAULT_DNS_TTL = 30;
```

Comment:
i know this is not a new constant introduced by this code, but why 30? a lot of aws services ive seen use at least 60. are we refreshing dns too much with default config?

Reply:
Assertion was incorrect about root reason.  host listener functionality was removed far earlier here: https://github.com/awslabs/aws-c-s3/pull/136

Will update PR commentary, but s3 client will be unaffected

Reply:
I don't what, if any, reasons were behind the original choice.